moped-database/views/component_arcgis_online_view.sql

```diff
@@ -114,22 +114,23 @@ related_projects AS (
     GROUP BY pmp.project_id
 ),
 
-min_phase_dates AS (
-    WITH min_dates AS (
-        SELECT
-            mpp.project_id,
-            min(mpp.phase_start) AS min_phase_start,
-            min(mpp.phase_end) AS min_phase_end
-        FROM moped_proj_phases mpp
-        LEFT JOIN moped_phases mp ON mpp.phase_id = mp.phase_id
-        WHERE mpp.is_phase_end_confirmed = false AND mpp.is_phase_start_confirmed = false AND mpp.is_deleted = false AND mp.phase_name_simple = 'Complete'::text
-        GROUP BY mpp.project_id
-    )
+latest_public_meeting_date AS (
+    SELECT
+        mpm.project_id,
+        coalesce(max(mpm.date_actual), max(mpm.date_estimate)) AS latest
+    FROM moped_proj_milestones mpm
+    WHERE mpm.milestone_id = 65 AND mpm.is_deleted = false
+    GROUP BY mpm.project_id
+),
 
+earliest_active_or_construction_phase_date AS (
```

Comment:
This one finds the earliest estimated or confirmed date of any phase with a simple name that is “Active” or “Construction”.